methods/fns.go

```diff
@@ -7,28 +7,42 @@ import (
 	"io/ioutil"
 	"log"
 	"net/http"
+	"strings"
 
 	"github.com/TylerBrock/colorjson"
 	"github.com/fatih/color"
+	"github.com/yosssi/gohtml"
 )
 
 // Formatresp formats the Response with Indents and Colors
 func formatresp(resp *http.Response) string {
+	var retbody string
+	heads := fmt.Sprint(resp.Header)
 	c := color.New(color.FgCyan, color.Bold)
 	magenta := color.New(color.FgHiMagenta)
 	yellow := color.New(color.FgHiYellow)
 	body, err := ioutil.ReadAll(resp.Body)
 	str := string(body)
-	var obj map[string]interface{}
-	json.Unmarshal([]byte(str), &obj)
-	f := colorjson.NewFormatter()
-	f.Indent = 6
-	s, _ := f.Marshal(obj)
-	for key, value := range resp.Header {
-		c.Print(key, " : ")
-		magenta.Print(value, "\n")
+	if strings.Contains(heads, "json") {
+		var obj map[string]interface{}
+		json.Unmarshal([]byte(str), &obj)
+		f := colorjson.NewFormatter()
+		f.Indent = 6
+		s, _ := f.Marshal(obj)
+		for key, value := range resp.Header {
+			c.Print(key, " : ")
+			magenta.Print(value, "\n")
+		}
+		retbody = yellow.Sprintf("\nStatus:\t\t%s\n\nStatusCode:\t%d\n", resp.Status, resp.StatusCode) + fmt.Sprintf("\n%s\n", string(s))
+	} else if strings.Contains(heads, "xml") || strings.Contains(heads, "html") {
```

Comment:
```suggestion
	} else if strings.Contains(heads, "xml") || strings.Contains(heads, "html") || strings.Contains(heads, "plain") {
```

Reply:
Thats a good catch. I don't know how I could missed that 😂. Possibly because of the Excitement I guess. Thanks for this 💟